instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Comment = props => {
+	return (
+		<div className='comment-section'>
+			<strong>{props.comment.username}</strong>{props.comment.text}
```

Comment:
Each `username` is running right up against the `comments text`. Adding a space or some other separator after the username would clean that up.